EIPS/eip-draft_tx_revert_protection.md

```diff
@@ -0,0 +1,228 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7640
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

Reply:
Updated, thank you.